programming_exercises/submit_from_git.rst

```diff
@@ -0,0 +1,54 @@
+Submit from Git
+===============
+
+A-plus supports exercises where student solutions are retrieved
+from their personal Git repository.
+
+.. warning::
+  
+  To test grading from Git, **a private RSA key MUST be appened to
+  the file at ``exercises/git_hello_python/git_rsa``.**
+  In May 2021, the access script in `grading-base`_ requires RSA key
+  that is used in cloning the repository via ssh URL.
```

Comment:
Is there an actual limitation that the key has to be RSA specifically or will any SSH privkey (rsa, ed25519, ecdsa, etc) work?

Reply:
The script in grading-base will use the key file in .ssh/ so anything that openssh & gitlab accepts as key pair should currently work.